tests/test_vis.py

```diff
@@ -340,7 +341,9 @@ def test_histogram_chart(global_var):
     lux.config.plotting_backend = "vegalite"
     vis = Vis(["Displacement"], df)
     vis_code = vis.to_Altair()
+    expected_bin_size = get_bin_size(vis.data["Displacement"])
```

Comment:
I still don't exactly understand this. `vis.data` is the data that has already been binned to 10 bins in the PandasExecutor, so in theory, the bin determination would need to go into the Compiler or `execute_binning` stage, if you want access to the raw data series (i.e.`ldf["Displacement"]`). Or is this simply an approximation that you've tested that works well in most cases?

Reply:
I took a look at `execute_binning` and I think I have an explanation (albeit an amateurish one). `execute_binning` successfully divides up the data into 10 bins using `np.histogram` and computes the centers where each of these bins would lie. However, what `execute_binning` fails to do is compute the bin widths. Thus, `get_bin_size` recomputes the bin width according to Freedman's Rule. 

But actually, this is a great point, I realized that `execute_binning` computes a secondary variable called `bin_edges`, which I could use to extract the bin width and store in the Vis object as `._bin_width`. In my most recent commit (#5b4643c), I experiment with this approach, and it yields the same results! Let me know if this is a preferred method, and I will update `test_vis.py` accordingly. @dorisjlee

Reply:
Thanks for the explanation @micahtyong! This makes a lot of sense. I was somewhat misled into thinking that `get_bin_size` computes the number of bins, instead of the bin width, which explains my confusion in thinking that this should go into the Executor. The original approach that you took seems like the more sensible approach. I'm glad that you were able to try this via the numpy `bin_edges` approach and verify that the results were similar. For the purpose of integrating with other types of executor (e.g. SQL), the bin width determination makes sense with the Freedman's Rule helper function that you had earlier.